x-pack/plugins/security_solution/public/dashboards/links.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { DASHBOARDS_PATH, SecurityPageName, SERVER_APP_ID } from '../../common/constants';
+import { DASHBOARD, DASHBOARDS } from '../app/translations';
+import type { LinkItem } from '../common/links/types';
+import { links as kubernetesLinks } from '../kubernetes/links';
+import { dashboardLinks as cloudSecurityPostureLinks } from '../cloud_security_posture/links';
+import {
+  ecsDataQualityDashboardLinks,
+  detectionResponseLinks,
+  entityAnalyticsLinks,
+  overviewLinks,
+} from '../overview/links';
+
+export const dashboardViewLinks: LinkItem = {
+  id: SecurityPageName.dashboardView,
+  title: DASHBOARD,
+  path: DASHBOARDS_PATH,
+  capabilities: [`${SERVER_APP_ID}.show`],
+  globalSearchKeywords: [
+    i18n.translate('xpack.securitySolution.appLinks.dashboard', {
+      defaultMessage: 'Dashboard',
+    }),
+  ],
+  sideNavDisabled: true,
+  // Dashboard view does not display as a landing card or icon
+  landingPageDisabled: true,
+  // Dashboard view needs url state
+  skipUrlState: false,
+  // Security / Dashboards / {Dashboard title}
+  skipBreadcrumb: true,
+};
```

Comment:
I don't think we need to define a new linkItem for the dashboardView page. The path (/dashboards) is already defined by the dashboardsLanding linkItem. We can use this one in the SpyRoute.
Actually, probably we don't even need to create the new SecurityPageName.dashboardView.

I understand that using SecurityPageName.dashboardsLanding for the new dashboard view is weird. But I think the problem is just the naming. We could probably rename it to just SecurityPageName.dashboards, completing the migration from landing_pages to the new dashboards sub-plugin 💯